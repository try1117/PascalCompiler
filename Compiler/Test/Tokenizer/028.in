begin{ multi
  line
  comment
}
  var abs 'qwerty'
end.
