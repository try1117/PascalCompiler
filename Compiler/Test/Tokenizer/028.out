0  | 0  | KEYWORD_BEGIN            | begin                    | 
4  | 2  | KEYWORD_VAR              | var                      | 
4  | 6  | VARIABLE                 | abs                      | 
4  | 10 | CONST_STRING             | qwerty                   | 
5  | 0  | KEYWORD_END              | end                      | 
5  | 3  | SEP_DOT                  | .                        | 
